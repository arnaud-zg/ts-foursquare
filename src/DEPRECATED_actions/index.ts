import { TLifeAction } from './life'

export type TRootAction = TLifeAction

export * from './life'
