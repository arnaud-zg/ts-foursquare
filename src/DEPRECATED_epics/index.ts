export * from './life'
export * from './lists'
export * from './photos'
export * from './venues'
