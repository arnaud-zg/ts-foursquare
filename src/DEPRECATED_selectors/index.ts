export * from './life'
export * from './photos'
export * from './request'
export * from './status'
export * from './venues'
