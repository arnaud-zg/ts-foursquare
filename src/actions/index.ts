import { TLifeAction } from './life'
import { TVenuesAction } from './venues'

export type TRootActionType = TVenuesAction | TLifeAction

export type TRootAction = TVenuesAction | TLifeAction

export * from './life'
export * from './venues'
