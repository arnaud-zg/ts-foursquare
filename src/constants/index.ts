export const PROJECT_NAME = 'ts-foursquare'
