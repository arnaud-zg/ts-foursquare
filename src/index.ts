require('isomorphic-fetch')

export * from './standalone'
export * from './utils'

export * from './DEPRECATED_actions'
export * from './DEPRECATED_reducers'
export * from './DEPRECATED_selectors'
