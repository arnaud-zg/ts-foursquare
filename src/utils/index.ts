export * from './icon'
export * from './url'
