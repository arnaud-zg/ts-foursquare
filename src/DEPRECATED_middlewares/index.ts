import { combineEpics, createEpicMiddleware } from 'redux-observable'
import { NStore } from '../../types'
import { TRootAction } from '../DEPRECATED_actions'
import {
  getAccessTokenEpic,
  getPhotosDetailsEpic,
  getVenuesCategoriesEpic,
  getVenuesExploreEpic,
  getVenuesLikesEpic,
  getVenuesListedEpic,
  getVenuesNextVenuesEpic,
  getVenuesSearchEpic,
  getVenuesSimilarEpic,
  getVenuesSuggestCompletionEpic,
  getVenuesTrendingEpic,
} from '../DEPRECATED_epics'

export const rootEpic = combineEpics(
  getAccessTokenEpic,
  getPhotosDetailsEpic,
  getVenuesCategoriesEpic,
  getVenuesExploreEpic,
  getVenuesLikesEpic,
  getVenuesListedEpic,
  getVenuesNextVenuesEpic,
  getVenuesSearchEpic,
  getVenuesSimilarEpic,
  getVenuesSuggestCompletionEpic,
  getVenuesTrendingEpic
)
export const epicMiddleware = createEpicMiddleware<
  TRootAction,
  TRootAction,
  NStore.IState
>()
