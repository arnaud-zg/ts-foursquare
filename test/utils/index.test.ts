import * as utils from '../../src/utils'

describe('utils', () => {
  it('should have a list of method', () => {
    expect(utils).toMatchSnapshot()
  })
})
