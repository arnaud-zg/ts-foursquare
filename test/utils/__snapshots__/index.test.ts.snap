// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils should have a list of method 1`] = `
Object {
  "getIconUrlList": [Function],
  "getLocationHref": [Function],
  "getLocationSearch": [Function],
}
`;
